<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>语料库选择</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="newCorpus">
      <property name="placeholderText">
       <string>select the corpus</string>
      </property>
      <property name="duplicatesEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QPushButton" name="uploadButton">
      <property name="text">
       <string>上传文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>172</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="keyword">
      <property name="placeholderText">
       <string>输入关键词</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>选择依赖关系</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="dep_key">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Select dependency factor</string>
      </property>
      <item>
       <property name="text">
        <string>complications</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>diagnostic guidelines</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>post conditions</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>配置API</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="advanceButton">
      <property name="text">
       <string>高级设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newCorpus</tabstop>
  <tabstop>uploadButton</tabstop>
  <tabstop>keyword</tabstop>
  <tabstop>dep_key</tabstop>
  <tabstop>advanceButton</tabstop>
  <tabstop>searchButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
